src/main/java/com/cinema/model/dto/MovieSessionResponseDto.java

```diff
@@ -5,6 +5,7 @@
 @Data
 public class MovieSessionResponseDto {
     private Long id;
+    private Long movieId;
```

Comment:
let's set this field in the mapper)

Reply:
I use ModelMapper to convert MovieSession to MovieSessionResponseDto. That's why I don't set it in the mapper.